src/rai_bench/rai_bench/tool_calling_agent/predefined/custom_interfaces_tasks.py

```
@@ -0,0 +1,88 @@
+    subtasks=[pub_HRIMessage_text_subtask]
+)
+get_interface_publish_ord_val = OrderedCallsValidator(
+    subtasks=[
+        get_HRIMessage_interface_subtask,
+        pub_HRIMessage_text_subtask,
+    ]
+)
+
+
+def get_custom_interfaces_tasks(
+    extra_tool_calls: List[int] = [0],
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)